### PR TITLE
Fix: packages updated

### PR DESCRIPTION
**Pull request by mwkyuen**
Paired with Zoe to get packages updated. Removed the Flex-Layout fork to use the latest version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,79 +3,68 @@
   "version": "0.0.0",
   "description": "Eplant",
   "main": "index.ts",
+  "type": "module",
   "scripts": {
-    "test": "jest --passWithNoTests",
-    "storybook": "start-storybook -p 6006",
-    "start": "vite serve",
-    "build": "tsc --noEmit && vite build",
-    "build-storybook": "build-storybook",
```
Comment (mwkyuen) [inline package.json:11]: Removed storybook deps

**Comment by zkdan** [comment]
With these files are you able to get ePlant up and running?

**Comment by mwkyuen** [comment]
Yes, able to install without issues and start the local instance, but `npm test` isn't working still.

File: jest.config.js
Comment (zkdan) [inline jest.config.js:1]: If we're removing storybook we can prob remove`    '^@stories/(.*)$': path.resolve(__dirname, 'stories/$1'),`, right?

File: package.json [hunk 2]
```diff
@@ -3,79 +3,69 @@
...
+  "type": "module",
   "scripts": {
     "test": "jest --passWithNoTests",
-    "storybook": "start-storybook -p 6006",
-    "start": "vite serve",
-    "build": "tsc --noEmit && vite build",
-    "build-storybook": "build-storybook",
-    "postinstall": "cd ./node_modules/flexlayout-react && npm install --legacy-peer-deps && npm run build:commonjs"
+    "dev": "vite",
+    "build": "tsc && vite build",
```
Comment (zkdan) [inline package.json:10]: Was [`--noEmit`](https://www.typescriptlang.org/tsconfig#noEmit) removed on purpose? If not, should we add it back into the build command?

> --noEmit: Do not emit compiler output files like JavaScript source code, source-maps or declarations.  This makes room for another tool like [Babel](https://babeljs.io/), or [swc](https://github.com/swc-project/swc) to handle converting the TypeScript file to a file which can run inside a JavaScript environment. You can then use TypeScript as a tool for providing editor integration, and as a source code type-checker.

**Review by zkdan — COMMENTED** [review]
Looks good! ~but when I copy-paste these files to my local, I can't get an instance running. I get an es-lint issue. I'm hoping we can merge these in and I can reclone and have what you have (i.e. a functional repo) (well except for the tests).~

**Review by Yukthiw — APPROVED** [review]
LGTM

**Pull request merged**